docs/PacketFence_PKI_Guide.asciidoc

```diff
@@ -207,15 +264,21 @@ Once your pki provider is configured we will create the provisioner configuratio
 
 Here is an example on how to configure an EAP-TLS connection for Windows/Android/MacOS/iOS
 
-image::docs/images/eaptls-example.png [scaledwidth="100%",alt="Provisioner EAP-TLS configuration"]
+image::docs/images/eaptls-example.png[scaledwidth="100%",alt="Provisioner EAP-TLS configuration"]
 
-+ explanation of screenshot
+* Provisioning ID: Name of your provisioner in PacketFence and of the profile on MacOS/iOS when installed
+* Roles: roles that will be affected by this provisioner
+* SSID: SSID to connect at after provisioning
+* Security type: You need to select the security type as *WPA2* to unlock access to *EAP type*, when you will chose *EAP-TLS* as an eap type, the PKI Provider will unlock
+* PKI Provider: chose the provider you configured earlier
 
 For MacOS/iOS you have a tab 'Signing' available, this tab will make your profile *Trusted* when trying to install on Apple device.
 
-image::docs/images/eaptls-sign-example.png [scaledwidth="100%",alt="Signing provisioner"]
+image::docs/images/eaptls-sign-example.png[scaledwidth="100%",alt="Signing provisioner"]
+
+To initiate the signing, you will need a Certificate deliver by a trusted provider(i.e. VeriSign, GoDaddy, etc...), the chain of certificate going with (i.e. gd-bundle.crt).
```

Comment:
Remove the ', the chain...' and add a line below :

You will also need the intermediate certificate if any.